### ISSUE TITLE
UI from rpm packcage does not start no fedora 37

### ISSUE THREAD

**Issue by successfully-failed** [freedomofpress/dangerzone#307]
An error in some python lib occured.

**Comment by successfully-failed** [comment]
Everything works after building from source

**Comment by deeplow** [comment]
@successfully-failed, in order for us to help, you need to be a bit more explicit.

After installing Dangerzone following the [official instructions](https://github.com/freedomofpress/dangerzone/blob/main/INSTALL.md#fedora), what happens when you run `dangerzone` in the terminal?

**Comment by apyrgio** [comment]
@successfully-failed: Just wanted to make sure, do you perhaps see this error?

```
importlib.metadata.PackageNotFoundError: No package metadata was found for dangerzone
```

If yes, you are probably affected by this bug https://github.com/freedomofpress/dangerzone/issues/156. We'll upload a new package version soon, which you should be able to install.

**Comment by apyrgio** [comment]
@successfully-failed: Probably this is the error you've encountered, since you can't do any action with Dangerzone after its installation.

We have published a new package (`dangerzone-0.4.0-2`) in the Fedora 37 repos, and we tested that this new package fixes the error you have encountered.

You can install this package either with `dnf update`, or `dnf install dangerzone` specifically.

I'll close this issue, since it seems to be resolved. If this is not the case, please send us an update. Thanks again for your report!
